src/utils/wallet-seed.js

```diff
@@ -109,5 +121,12 @@ async function deriveEncryptionKey(password, salt, iterations, digest) {
 }
 
 export function lockWallet() {
-  setUnlockedMnemonicAndSeed(null, null);
+  setUnlockedMnemonicAndSeed(null, null, null);
+}
+
+// Returns the 32 byte key used to encrypt imported private keys.
+async function deriveImportsEncryptionKey(seed) {
+  // SLIP16 derivation path.
+  return bip32.fromSeed(Buffer.from(seed, 'hex')).derivePath("m/10016'/0")
```

Comment:
@garywang any thoughts on the derivation path to use? https://wiki.trezor.io/Cryptocurrency_standards#SLIP16_-_Format_for_password_storage_and_its_encryption, seems reasonable, though not exactly the use case here, so not ideal. But I'm not sure making a new derivation path is any better?

Reply:
🤷‍♂️

Reply:
Hey @armaniferrante - my old GitHub is @garywang - wondering if you can help me. This is my Twitter will Dm you